tgui/packages/tgui/interfaces/Newscaster.js

```diff
@@ -343,6 +344,11 @@ const NewscasterFeed = (properties, context) => {
             <LabeledList.Item label="Owner">
               {channel.author || 'N/A'}
             </LabeledList.Item>
+            {!isAdmin && (
```

Comment:
why not just use `is_admin` here?

Reply:
Also it would need to be `!!is_admin`, because the result will be 1, then negated twice from false to true